tuning/config/configs.py

```diff
@@ -57,3 +57,7 @@ class TrainingArguments(transformers.TrainingArguments):
         default=False,
         metadata={"help": "Packing to be enabled in SFT Trainer, default is False"},
     )
+    tracker: str.lower = field(
+        default=None,
+        metadata={"help": "Experiment tracker to use. Requires additional configs, see tuning.configs/tracker_configs.py"}
```

Comment:
would be helpful to list the available options. from the code, looks like it would be "aim"

Reply:
done